connectors/mcp_debug_tool.py

```
@@ -3,543 +3,606 @@
-    
+
     Features:
     - Real-time code analysis and error detection
     - Quantum state debugging for quantum agents
     - GCP-powered reasoning and fix suggestions
     - Performance metrics and optimization insights
     - MCP-compliant context sharing
     """
-    
+
     def __init__(self, gcp_endpoint: str, auth_token: str = None):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)